### PR TITLE
Gh 288 one helm of gaa s

### PR DESCRIPTION
**Pull request by go159lab**
Currently the UI and Rest are deployed with separate helm charts. In order to simplify the install process make them both install from the same chart.
https://github.com/gchq/gaffer-experimental/issues/288

### CODE REVIEW

**Comment by t92549** [comment]
This puts everything in a top level `deploy` directory despite it only relating to gaas. Perhaps it would be a good time to move the ui inder `gaffer-as-a-service` as well? Then it can come under there?

**Comment by go159lab** [comment]
> This puts everything in a top level `deploy` directory despite it only relating to gaas. Perhaps it would be a good time to move the ui inder `gaffer-as-a-service` as well? Then it can come under there?

@t92549  I sent email about the semgrep issues in this PR

File: gaffer-as-a-service/deploy/helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Crown Copyright
```
Comment (GCHQDeveloper314) [inline gaffer-as-a-service/deploy/helm/values.yaml:1]: Should be 2022

File: gaffer-as-a-service/gaas-ui/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,17483 @@
+{
```
Comment (GCHQDeveloper314) [inline gaffer-as-a-service/gaas-ui/package-lock.json:1]: Assuming this file is required (it seems **very large**) - copyright header missing.

File: gaffer-as-a-service/gaas-ui/src/components/schema-builder/add-type.tsx [hunk 3]
```diff
@@ -91,6 +91,8 @@ export default function AddType(props: IProps): ReactElement {
     };
 
     function addTypeReducer(draft: any, action: any) {
+        const typeName = new RegExp("^[a-zA-Z]+$");
```
Comment (GCHQDeveloper314) [inline gaffer-as-a-service/gaas-ui/src/components/schema-builder/add-type.tsx:94]: Copyright header is missing, I expect this is the case for various other files as well.

File: gaffer-as-a-service/gaas-ui/package-lock.json (hunk 2 above)
Comment (K-itKat20) [reply gaffer-as-a-service/gaas-ui/package-lock.json]: removed

File: gaffer-as-a-service/gaas-ui/src/components/schema-builder/add-type.tsx (hunk 3 above)
Comment (K-itKat20) [reply gaffer-as-a-service/gaas-ui/src/components/schema-builder/add-type.tsx:109]: added missing copyright headers

File: gaffer-as-a-service/deploy/helm/values.yaml (hunk 1 above)
Comment (K-itKat20) [reply gaffer-as-a-service/deploy/helm/values.yaml]: updated

**Review by GCHQDeveloper314 — APPROVED** [review]
Some of the headers should have been 2021-2022, not 2022 only. This can be overlooked for now as Gaffer will be improved to introduce a plugin for automatically checking copyright headers.

**Comment by go159lab** [comment]
> headers should have been 2021-2022

Updated copyright headers to 2021-2022

**Comment by go159lab** [comment]
> 

Updated copyright headers to 2021-2022

File: gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/auth/JwtTokenUtilTest.java [hunk 4]
```diff
@@ -1 +1 @@
...
        given(intruder.getUsername()).willReturn("n3nad");

        //when
        String token = jwtTokenUtil.generateToken(authenticatedUser);
        boolean tokenValidated = jwtTokenUtil.validateToken(token, intruder);

        //then
        assertThat(tokenValidated).isFalse();
    }
}
```
Comment (GCHQDeveloper314) [inline gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/auth/JwtTokenUtilTest.java:1]: Seems to be an issue with this file, looks like line returns are missing entirely.

**Comment by t92549** [comment]
This now has merge conflicts.
Also if we  are moving the ui under gaffer-as-a-service, perhaps gaffer-as-a-service-tools can become gaffer-as-a-service/tools?

**Comment by rocky341** [comment]
Closing due to being replaced with #304

**Pull request closed without merging**